packages/@okta/courage-for-signin-widget/yarn.lock

```diff
@@ -584,9 +584,9 @@
     "@types/underscore" "*"
 
 "@types/eslint-scope@^3.7.0":
-  version "3.7.4"
-  resolved "https://artifacts.aue1e.internal:443/artifactory/api/npm/npm-okta-master/@types/eslint-scope/-/eslint-scope-3.7.4.tgz#37fc1223f0786c39627068a12e94d6e6fc61de16"
-  integrity sha1-N/wSI/B4bDlicGihLpTW5vxh3hY=
+  version "3.7.1"
```

Comment:
This version actually decreased, is that right?

Reply:
Yep, it looks like in the Gen3 branch we allowed the patch update but when I compared it to Gen2 (master branch), it was set to this earlier version so I accepted this change instead of upgrading to prevent issues.
See: https://github.com/okta/okta-signin-widget/blob/master/packages/%40okta/courage-for-signin-widget/yarn.lock#L586-L592